docs/guides/vpn/wireguard/client.md

```diff
@@ -51,11 +51,12 @@ For each new client, the following steps must be taken. For the sake of simplici
     Run the script like
 
     ```bash
+    chmod +x /path/to/script.sh
```

Comment:
If this is the preferred method of running the script then please change line 12 (shebang) to:

`#! /usr/bin/env bash`

Just in case `bash` isn't at the path coded in.